frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -97,6 +99,18 @@ export const FormFields = ({
     }, {})
   }, [augmentedFormFields, fieldPrefillMap])
 
+  // payment prefills - only for variable payments
+  if (searchParams.has(PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID)) {
+    const paymentParamValue = Number.parseInt(
+      searchParams.get(PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID) ?? '',
```

Comment:
```suggestion
      searchParams.get(PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID.toUpperCase()) ?? '',
```
btw it only works if I do `PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID.toUpperCase()` - should we make this case insensitive?

Reply:
my URL was http://localhost:3000/63f5e8b3fcdf2d0056a78e22?65562c526f8dd0004d4c58eb=hello&PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID=-100

Reply:
I'm thinking no. The short text prefill doesn't transform `toUpperCase` either. Thus, I think we should keep it consistent with each other lest admins question if they should uppercase the other prefill params.

Reply:
ok makes sense! can change the test description to be the value of the const `payment_variable_input_amount_field_id` instead of the constant's name

Reply:
Happy holidays!